serverless/src/submitHarmonyOrder/constructOrderPayload.js

```diff
@@ -215,6 +217,11 @@ export const constructOrderPayload = async ({
     orderPayload.append('outputCrs', selectedOutputProjection)
   }
 
+  // Adds supportsConcatenation to the payload and it's value
+  if (supportsConcatenation && enableConcatenateDownload) {
```

Comment:
I was having some issues running this branch locally and a job to harmony because `enableConcatenateDownload` was always undefined. I think there are a few places where that needs to be initialized. Its not being saved in the Database right now

Reply:
The change has been pushed up